pleskdistup/actions/systemd.py

```diff
@@ -75,3 +75,100 @@ def _post_action(self) -> action.ActionResult:
     def _revert_action(self) -> action.ActionResult:
         systemd.remove_systemd_service(self.service_name)
         return action.ActionResult()
+
+
+class RulePleskRelatedServices(action.ActiveAction):
```

Comment:
It's not clear what "rule" means here.

Reply:
The same thing as for se-linux actually. Let rename this action a little, maybe this will help